init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+:CheckOS
+IF "%PROCESSOR_ARCHITECTURE%"=="x86" (set bit=x86) else (set bit=x64)
```

Comment:
The Architecture is defined in the parameters to "build.cmd".  See [https://github.com/dotnet/cli/blob/rel/1.0.0/netci.groovy#L29](https://github.com/dotnet/cli/blob/rel/1.0.0/netci.groovy#L29) for an example (we use the same approach in VSO). This needed to read that parameter.